storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operations/get/PreparedGetObject.java

```diff
@@ -0,0 +1,237 @@
+package com.pushtorefresh.storio.sqlite.operations.get;
+
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.WorkerThread;
+
+import com.pushtorefresh.storio.StorIOException;
+import com.pushtorefresh.storio.sqlite.SQLiteTypeMapping;
+import com.pushtorefresh.storio.sqlite.StorIOSQLite;
+import com.pushtorefresh.storio.sqlite.queries.Query;
+import com.pushtorefresh.storio.sqlite.queries.RawQuery;
+
+import rx.Observable;
+
+import static com.pushtorefresh.storio.internal.Checks.checkNotNull;
+
+/**
+ * Created by ilyazorin on 17/11/15.
```

Comment:
Please remove this header, your commit will be in the git, so don't worry, nobody will forget your contribution :)